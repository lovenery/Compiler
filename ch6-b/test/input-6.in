(Fe(H3Cl4))3 -> Xe
