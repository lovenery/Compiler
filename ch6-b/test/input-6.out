Cl 12
Fe 3
H 9
Xe -1
